arcus/azureml/experimenting/resources/requirements.txt

```diff
@@ -0,0 +1,16 @@
+numpy
+pandas
+tqdm
+joblib
+matplotlib
+opencv-python
+seaborn
+argparse
+azureml-core
+arcus-azureml>=0.0.dev0
```

Comment:
- arcus-azureml>=0.0.dev0 is this correct?
- requirements file at this level is correct?